src/databricks/labs/remorph/transpiler/lsp/lsp_engine.py

```diff
@@ -375,7 +375,8 @@ async def _do_initialize(self, config: TranspileConfig) -> None:
         env = deepcopy(os.environ)
         for name, value in self._config.remorph.env_vars.items():
             env[name] = value
-        args = self._config.remorph.command_line[1:]
+        log_level = logging.getLevelName(logging.getLogger("databricks").level)
+        args = self._config.remorph.command_line[1:] + [f"--log_level={log_level}"]
```

Comment:
I'm wondering if this is the best way to signal `--debug` to pluggable transpilers, partly because it conflicts a little with the configuration file where the transpiler specifies the arguments it expects.

I think (at the least) we can do this more cleanly using a well-documented environment variable (eg. `REMORPH_DEBUG=1`).

(For more discussion/context, see [here](https://github.com/databrickslabs/morpheus/pull/229/files#r1987280058).)

Reply:
blueprint offers `is_in_debug` to translate the command line argument already.

Reply:
the transpiler command line knows nothing about `remorph`'s command line, and the purpose of the PR is to propagate the log level, not just the `--debug` flag.

Reply:
Please file a bug ? This PR is not about setting `remorph`'s log level

Reply:
Done https://github.com/databrickslabs/remorph/issues/1494

Reply:
@sundarshankar89  Can you approve the PR ?